.github/workflows/ci.yml

```diff
@@ -172,6 +172,10 @@ jobs:
       fail-fast: false
       matrix:
         type: [STD, RISCV]
+    env:
+      # TODO: (@davidsemakula) Update all manifests for integration-tests and remove this
```

Comment:
By this you mean adding the `abi = "…"` entry everywhere? I think that would be sensible, even if it's just for clarity. We should also add it to `cargo contract new`.